src/core_systems/jacobian.jl

```diff
@@ -19,7 +20,13 @@ For in-place systems, `jacobian` returns the Jacobian rule as a function
 at the state `u`, parameters `p` and time `t` and save the result in `J0`.
 """
 function jacobian(ds::CoreDynamicalSystem{IIP}) where {IIP}
-    _jacobian(ds, Val{IIP}())
+    if hasproperty(ds, :integ) &&
+            ds.integ.f isa SciMLBase.AbstractDiffEqFunction && !isnothing(ds.integ.f.jac)
```

Comment:
Sorry, I missed this before. There is the interface function: https://github.com/JuliaDynamics/DynamicalSystemsBase.jl/blob/main/src/core/dynamicalsystem_interface.jl#L138  can we use that function? from there `prob.f` should be the same function as `ds.integ.f`.